ui/components/imageWorkspace/src/components/SaveShape/SaveShapeForm.svelte

```diff
@@ -155,6 +155,22 @@
           </span>
           <Input
             type="number"
+            step="1"
```

Comment:
I guess you could use the same code and change only the step depending on type `int` or `float`